openapi_python_client/parser/properties/property.py

```diff
@@ -1,8 +1,27 @@
-from typing import ClassVar, Optional, Set
+from typing import ClassVar, NewType, Optional, Set
 
 import attr
 
-from ... import utils
+from ... import Config
+from ...utils import fix_keywords, fix_reserved_words, sanitize, snake_case
+
+_PythonIdentifier = NewType("_PythonIdentifier", str)
```

Comment:
Should this be underscored if it's being used by a bunch of other modules?

Reply:
I believe it's only being used directly by one other module. The `_` is really there to indicate that you shouldn't be constructing this class directly, but only by using the sanitization method. Best I could come up with for trying to enforce sanitization with types 🤷.

Although now that I'm thinking about it, maybe using a custom `__init__` would be better... what do you think?

Reply:
Nice, cool solution you ended up with.